templates/ceph/what-is-ceph.html

```diff
@@ -33,307 +33,344 @@ <h1>
+          </li>
+          <li class="p-list__item">
+            iSCSI Gateways (<code>ceph-iscsi</code>) that provide iSCSI targets for traditional block storage workloads such as VMware or Windows Server.
+          </li>
+        </ul>
+        <p>
+          Ceph stores data as objects within logical storage pools. A Ceph cluster can have multiple pools, each tuned to different performance or capacity use cases. In order to efficiently scale and handle rebalancing and recovery, Ceph shards the pools into placement groups (PGs). The CRUSH algorithm defines the placement group for storing an object and thereafter calculates which Ceph OSDs should store the placement group.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
 
-      <section class="p-strip--light">
-        <div class="row">
-          <div class="col-8">
-            <h2>
-              Ceph features
-            </h2>
-            <ul class="p-list__item is-split">
-              <li class="p-list__item is-ticked">
-                Thin provisioning of block storage for disk usage optimisation
-              </li>
-              <li class="p-list__item is-ticked">
-                Partial or complete read and writes and atomic transactions
-              </li>
-              <li class="p-list__item is-ticked">
-                Erasure coding for data protection
-              </li>
-              <li class="p-list__item is-ticked">
-                Snapshot history, cloning and layering support</li>
-                <li class="p-list__item is-ticked">
-                  POSIX file system semantics support
-                </li>
-                <li class="p-list__item is-ticked">
-                  Object level key-value mappings
-                </li>
-              </ul>
-            </div>
-          </div>
-        </section>
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>
+        Ceph features
+      </h2>
+      <div class="u-fixed-width">
+        <ul class="p-list__item is-split">
+          <li class="p-list__item is-ticked">
+            Thin provisioning of block storage for disk usage optimisation
+          </li>
+          <li class="p-list__item is-ticked">
+            Partial or complete read and writes and atomic transactions
+          </li>
+          <li class="p-list__item is-ticked">
+            Replication and erasure coding for data protection
+          </li>
+          <li class="p-list__item is-ticked">
+            Snapshot history, cloning and layering support
+          </li>
+          <li class="p-list__item is-ticked">
+            POSIX file system semantics support
+          </li>
+          <li class="p-list__item is-ticked">
+            Object level key-value mappings
+          </li>
+          <li class="p-list__item is-ticked">
+            Swift and AWS S3 Object API Compatibility
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
```

Comment:
The `div` with the class `row` needs to be closed before the `div` with the class `u-fixed-width` It makes a small difference. Please update it if you think it looks better! 
![image](https://user-images.githubusercontent.com/57550290/146185264-15af69e1-e528-4a35-bdd7-982e527779fb.png)
 
![image](https://user-images.githubusercontent.com/57550290/146184818-edee3dc7-878d-416a-8ebb-1086be06a492.png)

```suggestion
  <div class="row">
    <div class="col-8">
      <h2>
        Ceph features
      </h2>
    </div>
  </div>
  <div class="u-fixed-width">
    <ul class="p-list__item is-split">
        <li class="p-list__item is-ticked">
        Thin provisioning of block storage for disk usage optimisation
        </li>
        <li class="p-list__item is-ticked">
        Partial or complete read and writes and atomic transactions
        </li>
        <li class="p-list__item is-ticked">
        Replication and erasure coding for data protection
        </li>
        <li class="p-list__item is-ticked">
        Snapshot history, cloning and layering support
        </li>
        <li class="p-list__item is-ticked">
        POSIX file system semantics support
        </li>
        <li class="p-list__item is-ticked">
        Object level key-value mappings
        </li>
        <li class="p-list__item is-ticked">
        Swift and AWS S3 Object API Compatibility
        </li>
    </ul>
  </div>
```

Reply:
Done